README.md

```diff
@@ -13,63 +13,68 @@ validate, edit, and export SCAP documents.
 
 ## Compilation
 
-Choose *1a* or *1b* depending on whether you want sources from a release tarball or the git repository.
+Choose *2a* or *2b* depending on whether you want sources from a release tarball or the git repository.
 
-1a) Get a release tarball
+1) The following packages are required to be installed on your system:
+```
+autoconf automake libtool gcc
+```
+
+2) a) Get a release tarball and run ./autogen.sh:
 ```
 # replace ${version} with the desired version
 wget https://github.com/OpenSCAP/openscap/releases/download/${version}/openscap-${version}.tar.gz
 tar -xzpf openscap-${version}.tar.gz
 cd openscap-${version}
+./autogen.sh
```

Comment:
Is autogen.sh really needed on download tarball? I thought it was not mentioned here by purpose.